index.html

```diff
@@ -1299,11 +1316,11 @@ <h2>Verification Methods</h2>
         <dt>verificationMethod</dt>
         <dd>
 If a <a>DID document</a> includes a <code>verificationMethod</code> property,
-the value of the property MUST be an unordered set of verification methods,
-where each verification method is described by a set of properties.
-The set of properties MUST include the <code>id</code>, <code>type</code>,
-<code>controller</code>, and specific verification method properties, and
-MAY include additional properties.
+the value of the property MUST be a <a data-cite="INFRA#lists">list</a> of
```

Comment:
Should this be an ordered set instead? Same everywhere else that was converted from "set" => "INFRA list".

Reply:
Done in 1664a6fca40a0cf8a7df0fa7908609fb3ddabb16.